sections/marketplace/choosingyourpricingmodel.md

```diff
@@ -0,0 +1,39 @@
+---
+author: Veracity
+description: Choosing the right pricing model for your product can be challenging. Here we introduce some of the most common ones.
+---
+
+# Choosing your pricing model
+
+Choosing the right pricing model for your product can be challenging. There are several pricing models relevant for digital offerings and we will introduce some of the most common ones;​
+
+## Subscription model
+
+Subscription is a well-known model and it’s easily understandable for customers. A subscription model can be set up to be just one single subscription in which case you might not capture the entire revenue potential of your service, so using a tiered subscription is advised. When making a tiered subscription you should not make too many tiers as this will make it difficult for the customers to select the one most suitable for them.​
+
+Subscription models make the cost predictable for the customer and the revenue predictable for you as the provider of the service.​
+
+Some additional mechanisms can be considered when offering subscriptions; can you offer your customers a limited trial period? Why not let your customers try out what you can offer?​
+
+Setting the price for the top tier offering for large enterprises could be difficult so you might keep this tier negotiable and making it possible for potential customers to request a price quote here.
```

Comment:
Change all sentence "setting the price" to ---> "Price setting for the top tier (usually offered to large enterprises) could be difficult. You might keep this tier negotiable and leave the option for the potential customer to request a price quote."